### ISSUE TITLE
✅ 🐞 Bug Report: Swiper CSS overrides the display property of Containers with anchor tags - [ED-11293]

### ISSUE THREAD

**Issue by vladninja** [elementor/elementor#23087]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

### Description

.swiper-slide a class override the .e-con, .e-con > .e-con-inner with a display: inline, instead of getting --display variable when the container in a loop item is set to type "a (link)".

That especially cause a problem when the user have 2 elements inside container (which need to have a display flex), but instead of that container have an inline display, so items inside would not keep a flex behavior.
![div2](https://github.com/elementor/elementor/assets/76119088/2640385e-0cba-4dd0-a5f2-cf6dd17f10d9)
![a2](https://github.com/elementor/elementor/assets/76119088/0a7f6e57-f437-487d-826d-0d41b3fa289a)

### Steps to reproduce

1. Create a Loop Item
2. Set the container element type to a (link)
3. Place any content inside
4. Use carousel slider on any page with a created loop item for slider

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt

					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.54 (Debian)
	MySQL version: MySQL Community Server - GPL v8.0.31
	PHP Version: 7.4.33
	PHP Memory Limit: 
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: No
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: http://localhost:8083
	Home URL: http://localhost:8083
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==

	Elementor
		Version: 3.14.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.14.0
		Author: Elementor.com


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Active
	Hello Theme Header & Footer: Active
	Editor Top Bar: Inactive
	Landing Pages: Active
	Nested Elements: Active
	Nested Accordion: Inactive by default
	Lazy Load Background Images: Active
	Global Style Guide: Inactive by default
	Page Transitions: Active
	Notes: Active
	Loop: Active
	Form Submissions: Active by default
	Scroll Snap: Active
	Menu: Active


== Log ==
	


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

````

</details>

**Comment by nicholaszein** [comment]
Hello @vladninja! 

Thank you for submitting your issue! 🙏 

🏷️ We've labeled this issue and our team will take a look at it. If there is indeed a problem, they'll work on fixing it. We can't say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding!

🗒️ Note: Please, when submitting an issue, ensure you are using the latest version of Elementor, Elementor Pro, and the Hello Elementor theme, and that your website and server meet [Elementor's Requirements](https://elementor.com/help/requirements).

I can see that some of the requirements are not met:

```txt

== Server Environment ==
	PHP Memory Limit: ???

== WordPress Environment ==
	Memory limit: 40M => 256M+

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1 => 2.8.1+

== Active Plugins ==

	Elementor
		Version: 3.14.0 => 3.14.1+

	Elementor Pro
		Version: 3.14.0 => 3.14.1

```

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Meanwhile, a workaround is to add the following CSS to your Elementor `Editor` > `Site Settings` > `Custom CSS` section:

```css

.swiper-slide a.e-con {
    display: var(--display);
}

```

😉

**Comment by portrayaloflife** [comment]
Thank you for the workaround!

**Comment by nicholaszein** [comment]
Hello there @portrayaloflife!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
